lib/util/trip/drive_card.dart

```diff
@@ -72,6 +72,33 @@ class _DriveCardState extends TripCardState<Drive, DriveCard> {
     return SeatIndicator(trip);
   }
 
+  @override
+  Widget buildTopRight() {
+    if (!_fullyLoaded) {
+      return const Center(
+        child: SizedBox(),
+      );
+    } else if (_drive.cancelled) {
+      return Icon(
+        Icons.block,
+        color: statusColor,
+        key: const Key('cancelledIcon'),
+      );
+    } else if (_drive.rides!.any((Ride ride) => ride.status == RideStatus.pending)) {
+      return Icon(
+        Icons.access_time_outlined,
+        color: statusColor,
+        key: const Key('pendingIcon'),
+      );
+    } else {
+      return Icon(
```

Comment:
IMO, you don't actually need an icon in this case. We don't want to give the impression that the drive is somehow "finished".